W-18772199-pce-4.1-docs-kt (#372)
mulesoft/docs-private-cloud: modules/ROOT/pages/install-checklist.adoc

|===

=== Load Balancer Configuration

You can configure your load balancer to use any method for distributing client requests. Use a a round robin strategy for better results in most contexts. Make sure the load balancer is accessible through an IP address that is available to all machines on your network.

Make sure your load balancer routes these TCP ports:

[%header%autowidth.spread]
.TCP Port Routing Requirements
|===
|Load Balancer Port |Instance Port | Internal Usage
|`80` | `30080`  | HTTP redirects to HTTPs
|`443` | `30443` | HTTPS port
|`8083` | `30883` | HTTPS port for Mule runtimes to authenticate and renew certificates
|`8889` | `30889` | WebSocket port for Mule runtimes to connect
|===

In each case, configure your load balancer to listen on the load balancer port and redirect incoming requests to the instance port. Your Anypoint Platform PCE installation includes an internal NGINX server that listens on each of the configured instance ports, and then performs the action listed in the Internal Usage column.

Make sure your load balancer polls the address `HTTP:10256/healthz` to run a health check on your platform servers and confirm that they are accessible.

[NOTE]

> Do not configure SSL certificates in your load balancer. The platform handles the TLS termination with the certificates configured using Access Management. See xref:config-workflow.adoc[Configure Anypoint Platform PCE].

Reviewer: ```suggestion
Don't configure SSL certificates in your load balancer. The platform handles the TLS termination with the certificates configured using Access Management. See xref:config-workflow.adoc[Configure Anypoint Platform PCE].
```

* Access and connectivity to private container registry
* For OpenShift use SecurityContextConstraints for installing Anypoint Platform PCE.

=== Node Requirements for Anypoint Platform PCE 4.1

Before executing the `pcectl` binary for Anypoint Platform PCE 4.1, ensure you follow these requirements:

> * Use a Linux based operating system

Reviewer 2: We should add another requirement for gpg, like: Use gpg (GnuPG) version 2.1 or higher. Check the version by running: gpg --version

> * Use `ldd` (Linux Dynamic Loader) version 2.28 or higher +

Reviewer 3: ```suggestion
* The glibc version must be 2.28 or higher. Use `ldd` (Linux Dynamic Loader) utility tool to check the same.
```

* Must be secured over TLS.
* Must be accessible from installer nodes and Kubernetes clusters.
* Must be secured with either a username and password, or Certificate-based authentication. Make sure that the new or updated certificates are distributed to all nodes to facilitate uninterrupted TLS communication.

== Installer Node Requirements

These are the requirements:

* Operating system: RHEL, Ubuntu, or any Linux-based systems.
* Storage required a minimum 100GB.
* Access to Kubernetes cluster using kubeconfig.yaml
    ** The cluster needs `cluster:admin` access, as Anypoint Platform PCE installer needs to create resources of different types.

>     ** The cluster needs `read/write` access in the default namespace to run Anypoint Platform PCE installer jobs and other supporting services.

Reviewer 2: `the default namespace` should be updated to `the pce-core namespace`
